src/Microsoft.FeatureManagement/FeatureManager.cs

```diff
@@ -321,7 +321,7 @@ private void BindSettings(IFeatureFilterMetadata filter, FeatureFilterEvaluation
                     {
                         SlidingExpiration = ParametersCacheSlidingExpiration,
                         AbsoluteExpirationRelativeToNow = ParametersCacheAbsoluteExpirationRelativeToNow
-                    });
+                    }.SetSize(1));
```

Comment:
MemoryCacheEntryOptions has a size property. We should initialize it with the other properties rather than adding a new calling pattern.

```
,
                    new MemoryCacheEntryOptions
                    {
                        SlidingExpiration = ParametersCacheSlidingExpiration,
                        AbsoluteExpirationRelativeToNow = ParametersCacheAbsoluteExpirationRelativeToNow,
                        Size = 1
                    });
```